ngrinder-frontend/src/js/components/perftest/report/DetailReport.vue

```diff
@@ -61,6 +61,60 @@
                             <th v-text="i18n('perfTest.report.errors')"></th>
                             <td>{{ test.errors | numFormat }}</td>
                         </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.errorRate')"></th>
+                            <td>{{ test.errors /(test.tests + test.errors)*100 |  numFormat('0,0.00') }}
```

Comment:
You better clean up white spaces here like
```vue
<td>{{ test.errors / (test.tests + test.errors) * 100 | numFormat('0,0.00') }}
```

Reply:
LGTM, thanks. I will fix it.